runtime/flang/getvolinfo3f.c

```diff
@@ -8,27 +8,23 @@
 /*
  *	getvolinfo3f.c - DFWIN GetVolumeInformation subprogram
  *	This is just the interface to the actual system routine from
- *	the dwfin module routine.  This routine is called as an 'alias'
+ *	the dfwin module routine.  This routine is called as an 'alias'
  *	when the module is called; this routine makes sure that the
  *	output character arguments are blank-filled.
  */
 
 #include <string.h>
 
-typedef char *LPSTR;
-typedef int DWORD;
+#if defined (_WIN64)
```

Comment:
```suggestion
#if defined (_WIN32)
```
?

Reply:
Currently we only support Windows x64 and ARM64. That's why I used _WIN64 predefined macro instead of _WIN32.  _WIN32 defined as 1, when the compilation target is 32bit as well.

Reply:
even if only x64 win is supported shouldn't the constrain here not be as lose as possible? 
You want to know if the target is windows not if the target is x86 or x64. Basically I would use _WIN32 always if I want to know if something is Windows while I would use _WIN64 (or even better _M_X64) only if I need to differentiate between x64 and x86.

(Not that I care about x86)

Reply:
Yes, I can accept it :) I will update this.